lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -2579,6 +2580,7 @@ class ModuleWriter {
            "# if __has_include(<uchar.h>)\n"
            "#  include <uchar.h>\n"
            "# elif !defined(__cplusplus)\n"
+           "typedef char char8_t;\n"
```

Comment:
Nitpick: this should probably be `unsigned char`, since that's how it's [defined in the standard](https://en.cppreference.com/w/cpp/language/types). (Actual citation: C++20 [N4820](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/n4820.pdf) [basic.fundamental]p9.)

Reply:
Done.